ogb/lsc/mag240m.py

```diff
@@ -53,6 +54,25 @@ def download(self):
                 print('Stop download.')
                 exit(-1)
 
+    def to_pyg_hetero_data(self):
+        data = HeteroData()
+        path = osp.join(self.dir, 'processed', 'paper', 'node_feat.npy')
+        # Current is not in-memory
+        data["paper"].x = torch.from_numpy(np.load(path, mmap_mode='r'))
+        path = osp.join(self.dir, 'processed', 'paper', 'node_label.npy')
+        data["paper"].y = torch.from_numpy(np.load(path))
+        path = osp.join(self.dir, 'processed', 'paper', 'node_year.npy')
+        data["paper"].year = torch.from_numpy(np.load(path, mmap_mode='r'))
```

Comment:
We would need to add `data['author'].num_nodes = ...` and `data['institution'].num_nodes = ...` to register them as node types.

Reply:
data['author'].num_nodes = self.__meta__['author']
data['institution'].num_nodes = self.__meta__['institution']
I add these two lines to register `author` and `institution` as node types. And the RuntimeError is stil there.